dev/reloadable-server/src/main/scala/play/core/server/LagomReloadableDevServerStart.scala

```diff
@@ -83,6 +84,14 @@ object LagomReloadableDevServerStart {
           }
         }
 
+        val before = System.currentTimeMillis()
+        val address = InetAddress.getLocalHost
+        val after = System.currentTimeMillis()
+        if (after - before > 100) {
+          System.out.println(s"WARNING: Retrieving local host name ${address} took more than 100ms, this can create problems at startup with Lagom. \r\n" +
+            "You are probably using MACOS Sierra with a wrongly configured /etc/host . See https://thoeni.io/post/macos-sierra-java/ for a solution")
```

Comment:
Could we phrase it this way?

"If you are using macOS, see https://thoeni.io/post/macos-sierra-java/ for a solution."

I'm not sure if High Sierra has the same problem or not, or whether this will ever be fixed in future versions, or if it can affect other OSes, so it's a little more open.

Reply:
@TimMoore Might be pedantic, but I'd recommend phrasing it as "for a potential solution" as there's no guarantee the blog post will be a silver bullet.

Reply:
Yes, @erip good point.